whois-api/src/test/java/net/ripe/db/whois/api/httpserver/ClientCertificateServiceTestIntegration.java

```diff
@@ -26,6 +29,7 @@ public void client_certificate() {
     }
 
     @Test
+    @RetryFor(value = IOException.class, attempts = 10, intervalMs = 10000)
```

Comment:
Why is a retry needed?

Reply:
flaky tests, sometimes the server is not up and it return "unreachable server error"
This is now in master and it doesn't happen too often, but it happens

Reply:
OK. Please add a TODO comment, we could do something later like add a `Before` method to poll for the server coming up.

Reply:
Done